cli/command/cli.go

```diff
@@ -79,7 +79,9 @@ type DockerCli struct {
 	dockerEndpoint        docker.Endpoint
 }
 
-var storeConfig = store.NewConfig(
+// ContextStoreConfig contains the config used by the context store
+// Plugins init functions could modify it to define other typed endpoint metadata
```

Comment:
Nit: "A plugin's init functions may modify..."

Reply:
Less of a nit, but I think the sentence should end with something like "...by calling `ContextStoreConfig.SetEndpoint()`".